internal/quaiapi/quai_api.go

```diff
@@ -819,6 +819,44 @@ func (s *PublicBlockChainQuaiAPI) EstimateGas(ctx context.Context, args Transact
 	if blockNrOrHash != nil {
 		bNrOrHash = *blockNrOrHash
 	}
+
+	if args.from(s.b.NodeLocation()).IsInQuaiLedgerScope() && args.To.IsInQiLedgerScope() {
```

Comment:
Any special gas consideration for Qi -> Quai?

Reply:
Qi -> Quai doesn't use gas aside from ETX gas cost